### PR TITLE
Add Anki icon

### LINKED ISSUE #8723: Add Anki icon
### Brand Name

Anki

### Website

https://apps.ankiweb.net/

### Popularity Metric

The number of Github Stars is 14.1k. See https://github.com/ankitects/anki

### Official Resources for Icon and Color

svg: https://commons.wikimedia.org/wiki/File:Anki-icon.svg
color: #085cb4 (from Website Header)

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by uncenter**
anki

**Issue:** closes #8723.

**Popularity metric:** 16.9k stars (https://github.com/ankitects/anki).

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Hex color from the midpoint of the gradient of the bottom left star. Their website looks outdated so I didn't see any other source for the hex.

### CODE REVIEW

**Comment by uncenter** [comment]
There's also this option:

anki (1)

I think it looks way better (especially at small sizes) and is more recognizable, but since it isn't "accurate" to the project's *actual* logo would it not be fit for consideration?

**Comment by adamrusted** [comment]
Think we may be best to get someone from the Anki team (like @dae) to give some feedback on preferred approach here. How would you like the icon handled?

**Comment by dae** [comment]
The Anki logo is AGPL3, so if this is a derivative work, it will need a compatible license.

**Comment by adamrusted** [comment]
While this project is CC0 - we can (and do) list license details on a brand-by-brand basis, would that be suitable, @dae?
See our website:
image

And in our JSON:
https://github.com/simple-icons/simple-icons/blob/4e4a29e74e2f67ad2e32bcb36ce378d0e55101e5/_data/simple-icons.json#L398-L405

**Comment by dae** [comment]
If it's clearly marked as AGPL, no objections from me.

**Comment by adamrusted** [comment]
Fab, thanks @dae - last question, which of @uncenter's treatments of the icon do you prefer?
image
image

**Comment by dae** [comment]
The former is closer to the status quo, but looks a bit much when it's placed in something with its own frame like a button. I think it really depends on the context it's going to be used in.

**Comment by adamrusted** [comment]
Most common usage (our main dependent) is shields.io - though users often use the package for their own websites, etc. I would think our general max size is <128px tall. @dae

**Comment by dae** [comment]
Then maybe the latter, but I don't really have a preference.

**Comment by adamrusted** [comment]
Thanks for the guidance, @dae - I've now merged the license and 'guidelines' (as mentioned on in the LICENSE file). @uncenter - would you be able to merge the latter treatment into this PR an I'll review it?

File: icons/anki.svg
Comment (adamrusted) [inline icons/anki.svg:1]: @uncenter - I can see _some differences_ when comparing, either against the stars with the outline, or without. Could you advise how you created this variant, and I'll do the same on my end.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/anki.svg
Comment (uncenter) [reply icons/anki.svg:1]: > Could you advise how you created this variant, and I'll do the same on my end.

First I clear out all the extra symbols/shapes from the source icon, leaving just the logo with the two stars. I then delete everything in this shape as well, leaving just the stars themselves. I lock the aspect ratio, resize them to 24x24, and center them on the page. Then I convert the strokes to paths, fill in the inner part of the stars (fill tool + union) and finally merge the two stars together.

Comment (adamrusted) [reply icons/anki.svg:1]: Just copying the stars out of source and resizing to 24px high, I'm still seeing differences:
image

Comment (uncenter) [reply icons/anki.svg:1]: This is so strange! Everything lines up in my editor, and then I save it, reopen it, and it no longer lines up :/

Comment (adamrusted) [reply icons/anki.svg:1]: I blame @PeterShaggyNoble - he's usually the cause for my SVG woes 🤣

Comment (uncenter) [reply icons/anki.svg:1]: Now for some reason my duplicate of the original stars that I left in to compare with aren't lining up with... the original stars once I import the original SVG file again. Arghhh what is going on!?

**Review by uncenter — COMMENTED** [review]
New path looks way better! Thanks @adamrusted - I can't approve but LGTM :)

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**